MerlinAU.sh

```diff
@@ -4828,38 +5029,48 @@ _ShowNodesMenuOptions_()
     done
 }
 
-##---------------------------------------##
-## Added by ExtremeFiretop [2024-May-02] ##
-##---------------------------------------##
+##----------------------------------------##
+## Modified by Martinski W. [2024-May-04] ##
+##----------------------------------------##
 _DownloadChangelogs_()
 {
+    local wgetLogFile  changeLogTag  changeLogFile  changeLogURL
+
     changeLogTag="$(echo "$(nvram get buildno)" | grep -qE "^386[.]" && echo "386" || echo "NG")"
     if [ "$changeLogTag" = "386" ]
     then
-        wget -O "$FW_BIN_DIR/Changelog-${changeLogTag}.txt" "${CL_URL_386}"
+        changeLogURL="${CL_URL_386}"
     elif [ "$changeLogTag" = "NG" ]
     then
-        wget -O "$FW_BIN_DIR/Changelog-${changeLogTag}.txt" "${CL_URL_NG}"
+        changeLogURL="${CL_URL_NG}"
     fi
+
+    wgetLogFile="${FW_BIN_DIR}/${ScriptFNameTag}.WGET.LOG"
     changeLogFile="${FW_BIN_DIR}/Changelog-${changeLogTag}.txt"
+    printf "\nRetrieving ${GRNct}Changelog-${changeLogTag}.txt${NOct} ...\n"
+
+    wget --timeout=5 --tries=4 --waitretry=5 --retry-connrefused \
+         -O "$changeLogFile" -o "$wgetLogFile" "${changeLogURL}"
+
     if [ ! -f "$changeLogFile" ]
     then
         Say "Change-log file [$changeLogFile] does NOT exist."
+        echo ; [ -f "$wgetLogFile" ] && cat "$wgetLogFile"
```

Comment:
Since we changed the method to "more" from "less" here in my PR.
I notice that it's now uppercase Q to close straight away, instead of lowercase q.

I realize this is not a you change, it was changed in my PR with your suggestion, but if I could have you modify this line so it properly reflects the new updated closure process with "more". Something like this:

        printf "\nPress uppercase '${REDct}Q${NOct}' to quit when finished.\n"